TestFoundation/TestNSString.swift

```diff
@@ -1039,8 +1039,8 @@ func checkHasPrefixHasSuffix(lhs: String, _ rhs: String, _ stack: [UInt]) -> Int
     }
 
     var failures = 0
-    failures += testFailure(expectHasPrefix, lhs.hasPrefix(rhs), stack + [__LINE__])
-    failures += testFailure(expectHasSuffix, lhs.hasSuffix(rhs), stack + [__LINE__])
+    failures += testFailure(expectHasPrefix, lhs.hasPrefix(rhs), stack + [#line])
```

Comment:
Seems `[#line]` doesn't compile but `[ #line] does.  Intentional as part of SE-0028?